MANIFEST.in

```diff
@@ -5,6 +5,7 @@ include chaco/tools/toolbars/images*.png
 include chaco/tools/toolbars/images*.svg
 include chaco/tools/toolbars/images*.txt
 include chaco/layers/data/*.svg
+recursive-include chaco *.pyx
```

Comment:
Am i missing something or should we be including the `cntr.c` file?
#459 was doing this.

Reply:
C files that are listed in extension modules are included by default (https://packaging.python.org/guides/using-manifest-in/). I missed that in #459.